### DISCUSSION TITLE
Inability to delete a repository

### DISCUSSION THREAD

**Discussion by NatashkaJK** [community/community#121215] [Repositories]
### Select Topic Area

Bug

### Body

I have a problem,when the repos is using all my disc C,like:
![image](https://github.com/community/community/assets/158469400/2ea92f30-315c-4958-bc2e-c9897c6c2ec3)
and i can't delete.because
![image](https://github.com/community/community/assets/158469400/18819616-cb67-4675-b2bc-2996c32b612c)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @NatashkaJK !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Repositories category. I've gone ahead and moved it for you. Good luck!
